features/shadow/distsql/handler/src/test/java/org/apache/shardingsphere/shadow/distsql/update/DropDefaultShadowAlgorithmStatementUpdaterTest.java

```diff
@@ -55,12 +59,15 @@ public void assertCheckWithIfExists() {
     
     @Test
     public void assertUpdate() {
+        String algorithmName = "default";
         ShadowRuleConfiguration ruleConfig = new ShadowRuleConfiguration();
-        ruleConfig.setDefaultShadowAlgorithmName("default");
+        ruleConfig.getShadowAlgorithms().put(algorithmName, mock(AlgorithmConfiguration.class));
+        ruleConfig.setDefaultShadowAlgorithmName(algorithmName);
         DropDefaultShadowAlgorithmStatement statement = new DropDefaultShadowAlgorithmStatement(false);
         updater.checkSQLStatement(database, new DropDefaultShadowAlgorithmStatement(true), ruleConfig);
         assertTrue(updater.hasAnyOneToBeDropped(statement, ruleConfig));
         updater.updateCurrentRuleConfiguration(statement, ruleConfig);
         assertNull(ruleConfig.getDefaultShadowAlgorithmName());
+        assertThat(ruleConfig.getShadowAlgorithms().size(), is(0));
```

Comment:
`assertTrue(ruleConfig.getShadowAlgorithms().isEmpty());` will be better?

Reply:
> `assertTrue(ruleConfig.getShadowAlgorithms().isEmpty());` will be better?

ok